### PR TITLE
applications: asset_tracker_v2: Tweak transmission buffers

### PR DESCRIPTION
**Pull request by simensrostad**
Tweak transmission buffers to reduce the memory footprint of the
lwm2m build of the application.

With CBOR enabled the total size of messages has been reduced, this
allows us to set the max size of CoAP messages lower.

### CODE REVIEW

File: applications/asset_tracker_v2/overlay-lwm2m.conf [hunk 1]
```diff
@@ -6,14 +6,19 @@ CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
 CONFIG_LWM2M_RW_OMA_TLV_SUPPORT=y
 CONFIG_LWM2M_VERSION_1_1=y
 CONFIG_LWM2M_ENGINE_STACK_SIZE=3072
-CONFIG_LWM2M_COAP_MAX_MSG_SIZE=2304
```
Comment (jheiskan81) [inline applications/asset_tracker_v2/overlay-lwm2m.conf:9]: Now You use 512 only?. Set 1280.

Comment (simensrostad) [reply applications/asset_tracker_v2/overlay-lwm2m.conf:9]: It defaults to the CoAP block size which is set to `1024`.

Comment (simensrostad) [reply applications/asset_tracker_v2/overlay-lwm2m.conf:9]: Any particular reason why it should default to `1280`?

Comment (jheiskan81) [reply applications/asset_tracker_v2/overlay-lwm2m.conf:9]: OK if everything is working then it is fine.

**Review by jheiskan81 — APPROVED** [review]

**Review by maxd-nordic — APPROVED** [review]

**Pull request merged**